example/src/main/config/hono-adapter-http-vertx-config.yml

```diff
@@ -33,6 +33,14 @@ hono:
     port: 5671
     credentialsPath: ${secret.path}/http-adapter.credentials
     trustStorePath: ${secret.path}/trusted-certs.pem
+  command:
+    name: 'Hono HTTP Adapter'
+    amqpHostname: hono-http-internal
```

Comment:
Wouldn't `hono-http-command` be a better name?

Reply:
My intention was that this is the connection from the adapter to the AMQP 1.0 network, which is at the moment used for command and control.